.semaphore/bangladesh_demo_deployment.yml

```diff
@@ -0,0 +1,25 @@
+version: v1.0
+name: Pipeline 3
+blocks:
+  - name: Deploy to Bangladesh Demo
+    task:
+      jobs:
+        - name: Deploy to Bangladesh Demo
+          commands:
+            - checkout
+            - sem-version ruby 2.5.1
+            - cache restore
+            - yarn install
+            - bundle install --deployment --path vendor/bundle
+            - cache store
+            - 'BRANCH=$SEMAPHORE_GIT_BRANCH bundle exec cap bangladesh:demo deploy'
```

Comment:
I'm guessing semaphore has some way to DRY up these deployment commands somehow, so we don't need to repeat them for each environment.  Not a huge deal but would be nice.

Reply:
Actually, I'm guessing we can just create a `bin/sem_deploy` that does all of them perhaps? They are just shell commands run on semaphore's ec2 containers I believe, so that would probably work.